src/conv.rs

```diff
@@ -1,7 +1,8 @@
-use crate::{make_slice, map_enum, native, Label, OwnedLabel};
+use crate::{make_slice, map_enum, native, Label, OwnedLabel, follow_chain};
 use naga;
 use std::{borrow::Cow, ffi::CStr, num::NonZeroU32};
 use wgc::{id, pipeline::ShaderModuleSource};
+use wgt::{PushConstantRange};
```

Comment:
let's not include this. See how all the code here just refers to the fully qualified type. So this would be `wgt::PushconstantRange`